internal/cluster/cluster.go

```diff
@@ -0,0 +1,15 @@
+package cluster
+
+type Cluster struct {
+	OrganizationSlug string
+	ClusterID        string
+	Queues           []Queue
+	Name             string
+	Description      string
+}
+
+type Queue struct {
+	Id    string
+	Name  string
+	Agent int
```

Comment:
Is this the number of agents currently registered in the queue? I think we should name it more along the lines of that if it is

Reply:
Yes it is registered and running agents. Should I rename it to Active Agents?

Reply:
Maybe something like connected agent count?